app/models/dwp_override.rb

```diff
@@ -1,2 +1,5 @@
 class DWPOverride < ApplicationRecord
+  belongs_to :legal_aid_application
+
+  validates :passporting_benefit, :evidence_available, presence: true
```

Comment:
These two attributes are added across two pages according to the design. Can we rely on separated validations in the model for each of these pages? Or leave just `validates :passporting_benefit` here and then update with evidence available on the 2nd step of the dwp override flow?